cmd/kops/rollingupdatecluster.go

```diff
@@ -264,5 +271,27 @@ func RunRollingUpdateCluster(f *util.Factory, out io.Writer, options *RollingUpd
 		return nil
 	}
 
-	return d.RollingUpdate(groups, k8sClient)
+	if featureflag.ValidiateAndDrainRollingUpdate.Enabled() {
+		d := &kutil.RollingUpdateClusterDV{
+			MasterInterval: options.MasterInterval,
+			NodeInterval:   options.NodeInterval,
+			Force:          options.Force,
+			K8sClient:      k8sClient,
+			ForceDrain:     options.ForceDrain,
+			FailOnValidate: options.FailOnValidate,
+			CloudOnly:      options.CloudOnly,
+			ClusterName:    options.ClusterName,
+			Cloud:          cloud,
+		}
+		glog.V(2).Infof("New rolling update with drain and validate enabled")
```

Comment:
I will look at what the rest of the code is doing, and match that.  We may be doing it incorrectly, but it would be better to do it consistently.